variables.tf

```diff
@@ -14,6 +14,18 @@ variable "instance_name" {
   description = "Name of the security and compliance instance that will be provisioned by this module"
 }
 
+variable "attach_wp_to_scc_instance" {
+  type        = string
+  description = "Set to `true` to attach an existing SCC Workload Protection instance to attach it to the SCC instance."
```

Comment:
When set to true, a value must be passed for the `wp_instance_crn` inout variable